docker/sawtooth-base

```diff
@@ -21,6 +21,7 @@ RUN apt-get update && apt-get install -y -q \
     python3-pip \
     python3-colorlog \
     python3-cbor \
+    python3-requests \
```

Comment:
Is the `python3-requests` package something that will be used by all components, or just the validator? If it is just the validator, then it should be added to `docker/sawtooth-validator`. Otherwise this is fine.